doc/api/http.markdown

```diff
@@ -103,7 +103,7 @@ of these values set to their respective defaults.
 To configure any of them, you must create your own [`http.Agent`][] object.
 
 ```javascript
-var http = require('http');
+const http = require('http');
```

Comment:
Looking it over, I agree. It's not a huge chunk of work. Already have it
mostly done. Will update this PR with the other updates.
On Dec 14, 2015 4:51 PM, "Colin Ihrig" <notifications@github.com> wrote:

> In doc/api/http.markdown
> <https://github.com/nodejs/node/pull/4282#discussion_r47583137>:
>
> > @@ -103,7 +103,7 @@ of these values set to their respective defaults.
> >  To configure any of them, you must create your own [`http.Agent`][] object.
> >
> >  ```javascript
> > -var http = require('http');
> > +const http = require('http');
>
> It might actually be better to have huge docs only churn in a single
> commit, instead of 32 commits <https://github.com/nodejs/node/pull/3662>.
> People are a lot less likely to bisect and blame the docs.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/nodejs/node/pull/4282/files#r47583137>.
>